src/components/Layout.js

```diff
@@ -3,7 +3,7 @@ import Navbar from './Navbar';
 
 const Layout = (props) => {
     return (
-        <div>
+        <div className="bg-gray-50">
```

Comment:
Instead of it being a component, it can be defined as a `container` or `page`,  e.g `HomePage.js`. Components as terms says mean something which is modular and can be reused.

Reply:
I see home page is already there. Don't see a need for this component.